documentation-site/components/yard/config/breadcrumbs.ts

```diff
@@ -0,0 +1,42 @@
+import {Breadcrumbs} from 'baseui/breadcrumbs';
+import {StyledLink} from 'baseui/link';
+import {PropTypes} from '../const';
+import {TConfig} from '../types';
+
+const BreadcrumbsConfig: TConfig = {
+  imports: {
+    'baseui/breadcrumbs': {
+      named: ['Breadcrumbs'],
+    },
+  },
+  scope: {Breadcrumbs, StyledLink},
+  theme: ['breadcrumbsText', 'breadcrumbsSeparatorFill'],
+  props: {
+    children: {
+      value: `
+        <StyledLink href="#basic-parent">Parent Page</StyledLink>
+        <StyledLink href="#basic-subparent">Sub-Parent Page</StyledLink>
+        <span>Current Page</span>
+      `,
```

Comment:
Could you break this into more lines and also remove the first and last empty row? Currently, the initial knob value is not going through prettier, so it looks like this.

<img width="322" alt="Screen Shot 2019-10-03 at 2 21 16 PM" src="https://user-images.githubusercontent.com/1387913/66165241-33295c00-e5e9-11e9-92a5-799faf0a8049.png">

I might add prettier even during initialization, was just trying to avoid it to keep it faster.